client/dashboard/summary-button/style.scss

```diff
@@ -0,0 +1,65 @@
+.summary-button {
```

Comment:
Feels like there's a lot of CSS, maybe our components don't compose well?

Reply:
I wouldn't say much TBH.. We need the first lines to override the `Button` styles when we have an `a` tag and some colors on hover, etc..

Also regarding the `Text` usages and rules for `summary-button-title` for example, I think it's fine to add a line of css and get the benefits of `Text`.

Reply:
I think CSS within the reusable components is fine, so I'm not too concerned. I believe we should try to avoid CSS more at the app level but still something to pay attention to.